packer/github-actions-runner/build.pkr.hcl

```diff
@@ -47,6 +47,20 @@ build {
     ]
   }
 
+  provisioner "file" {
+    content = templatefile("./start-runner.sh", {
+      start_runner = templatefile("../start-runner.sh", { metadata_tags = "enabled" })
```

Comment:
Why does the provisioner reference the start-runner.sh script twice here? And why is the second in the parent directory?

Reply:
Oh I see it's because the aws-github-runner repo uses the first as a wrapper script. I think we could drop the wrapper and just have a single template file.